index.html

```diff
@@ -950,11 +956,253 @@ <h3>
+        </h5>
+        <p>
+          The outlined risks of government-issued digital credentials present a
+          challenge that can not be solved by a single participant in the
+          ecosystem, and will require a broader society-wide discussion about
+          the risks and benefits of accessing online services through real
+          world credentials.
+        </p>
+        <p>
+          It is desirable that a government that issues digital credentials
+          also provides laws and regulations which clearly define how and for
+          which purposes those credentials are able to be used. All parties
+          involved in the exchange, whether they are legally obliged to do so
+          or not, are advised to support any government verifier authentication
+          schemes, if they exist. The support for (and integration of) verifier
+          authentication schemes such as <a href=
+          "https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-2/annex-2-high-level-requirements.md#a2327-topic-27---registration-of-pid-providers-providers-of-qeaas-pub-eaas-and-non-qualified-eaas-and-relying-parties">
+          EUDI access and registration certificates</a> can mitigate risks of
+          proliferation of unnecessary credential requests. However, the
+          presence of such schemes is not guaranteed, which significantly
+          increases the risk in a credential exchange.
+        </p>
+        <p>
+          There are other practical steps that user agents implementing the
+          Digital Credentials API can take to reduce risk, increase user
+          understanding and prevent certain types of harm.
+        </p>
+        <ul>
+          <li>Requiring protocols to support selective disclosure and other
+          techniques for data minimization can reduce the impact and likelihood
+          of information leakage, and provide better context to users in
+          permission and consent flows.
+          </li>
+          <li>Requiring protocols to support unlinkability mechanisms such as
+          Zero-Knowledge Proofs can prevent verifier-based surveillance, and
+          potential discrimination by hiding the issuer.
+          </li>
+          <li>Offering useful context and a clearly understandable permission
+          flow will help users make better decisions on whether or not to
+          accept a credential exchange, which can reduce the viability of
+          exchange requests that are made without a concrete user need.
+          </li>
+        </ul>
+        <p>
+          It is further critical that user agents design a permission
+          experience that accounts for the lack of these mitigations, e.g. the
+          exchange of personal information from government credentials without
+          any verifier authentication scheme. It is recommended to apply a very
```

Comment:
I think I'd say "higher level of friction". It's not yet clear to me if the ecosystem is going to converge on using verifier authentication at scale, and "very high level of friction" suggests to me something almost unusable (like what we do for safebrowsing warnings where we're sure a page is phishing). But a key principle that resonates with me is that the UA should differentiate higher and lower risk scenarios to help users in deciding how to engage with such UI.

Reply:
That works for me!